update_signal_collection.sh

```diff
@@ -0,0 +1,3 @@
+PROXY="http://localhost:6764"
+APP="YOUR_FUSION_APP_ID"
+curl -u $CREDS -X POST "$PROXY/api/update/all/signalCollection" > ${APP}_signal_collection_update.json
```

Comment:
why do we need to have this script as a separate file and also have its contents duplicated in the README, should it be either one thing or another?

Reply:
That's how I have written the instruction.. either execute those or run this script with the creds.
Is it fine? Do we need to have remove the script